### DISCUSSION TITLE
Welcome to Inpaint-Anything Discussions!

### DISCUSSION THREAD

**Discussion by geekyutao** [geekyutao/Inpaint-Anything#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elanwy** [comment]
Using masks to generate images with glowing strokes on the edges

**Comment by motowuyong** [comment]
MAN

**Comment by harilachu** [comment]
Is it possible to replace a text in an image with another one meeting the text bounds and also without affecting the background?

**Comment by 1960-dot** [comment]
I have the problem that after segmentation I cannot mark or paint the inpaint areas
